### PR TITLE
Add Thread TimeStamp to Msg

### PR DESCRIPTION
**Pull request by Davejkane**
Thread time stamp is required to determine if a message received is
received in a thread or in a channel proper.

### CODE REVIEW

File: messages.go [hunk 1]
```diff
@@ -17,15 +17,16 @@ type Message struct {
...
-	IsStarred   bool         `json:"is_starred,omitempty"`
-	PinnedTo    []string     `json:"pinned_to, omitempty"`
-	Attachments []Attachment `json:"attachments,omitempty"`
-	Edited      *Edited      `json:"edited,omitempty"`
+	Type            string       `json:"type,omitempty"`
+	Channel         string       `json:"channel,omitempty"`
+	User            string       `json:"user,omitempty"`
+	Text            string       `json:"text,omitempty"`
+	Timestamp       string       `json:"ts,omitempty"`
+	ThreadTimeStamp string       `json:"thread_ts,omitempty"`
```
Comment (dlsniper) [inline messages.go]: To keep this consistent with the other ` Timestamp ` field that exists already, can you please change it to ` ThreadTimestamp `?

**Review by dlsniper — CHANGES_REQUESTED** [review]
Hello,

Thank you for sending this PR.
There's a small change that should be done in order to keep the field name consistency.
Otherwise, this looks good.

Thank you.

**Comment by Davejkane** [comment]
Hi,

The requested changes to the name of ThreadTimestamp have been made.

Cheers

**Review by dlsniper — APPROVED** [review]
Thank you, looks good to me.

**Comment by dlsniper** [comment]
Hi @Davejkane, 

Sorry for the delay in merging this.
Thank you so much for the change. Looks good.

**Pull request merged**